app/lib/zooniverse-logging.js

```diff
@@ -37,6 +37,10 @@ class ZooniverseLogging {
     this.adapters = this.adapters.concat(newAdapters);
     this.adapters.forEach(adapter => adapter.configure(this.keys));
   }
+
```

Comment:
do we need to add `zooniverse/snapshot-wisconsin` to the `enabledTokens`? 

I think filtering by project can be handled by GTM or GA, and when I test classifying and going to Talk locally it's logging events regardless of that fact that I'm not in one of the specific projects noted in `enabledTokens` because it still has `zooHome` as the `projectToken`.

so I don't think so, but want to double-check. and not sure if logging things from every project is an issue? doesn't appear like the `projectToken` is being set by project.

Reply:
Note this was addressed by @mcbouslog in #4650